tests/dat/deploy1.yaml

```diff
@@ -1,4 +1,6 @@
 application:
   name: wskdeploy-samples
   namespace: /wskdeploy/samples/
+  credential: 23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP
```

Comment:
@paulcastro that key is not sensitive, it's the default key for user `guest` from openwhisk core repo used for testing.
https://github.com/openwhisk/openwhisk/blob/master/ansible/files/auth.guest